ffi/src/installer.rs

```diff
@@ -141,6 +141,42 @@ pub unsafe extern "C" fn distinst_installer_on_status(
 
 /// Install using this installer
 #[no_mangle]
+pub unsafe extern "C" fn distinst_installer_install_and_retain_home(
```

Comment:
Perhaps retain_home should be added to the config?

Reply:
I'll try that